src/helpers.ts

```diff
@@ -7,3 +12,32 @@ export const displayIdentity = (address: string, chainId?: number) => {
     default: return addr
   }
 }
+
+/**
+ * Create Identity
+ * @param address address for the did
+ * @param chainId chainId
+ * @param includeEthr include eth:, required for some did tools
+ */
+export const createDidFormat = (address: string, chainId: number, includeEthr?: boolean) => {
```

Comment:
We can use `@rsksmart` version to set did method from the `EthrDID` lib instead of handling it manually. See https://github.com/rsksmart/ethr-did/pull/1/files#diff-6ed28c397a4ab046b632d995deccdd429870fec30bb17b2c7781ae986aa6d4afR65

Reply:
I need guidance here on when we should show an address vs a did.

Reply:
Mmh. The resolver is doing output of wrong DID method network in the DID field?

Reply:
Perhaps I am using it wrong then. This is how I implemented the resolver: https://github.com/rsksmart/rif-identity-manager/pull/5/files#diff-156fa126352dd17b9b3813d3ed3840533b4201af079451c3702461bc9d41f667R50-R57

Should I be getting something from EthrDid first?

Reply:
Yes, we can pass `method` param to `EthrDID`. That will prevent the need of converting each time the DID is displayed. That would imply `method: getDIDMethod(chainId)` just in that configuration you are pointing.

Reply:
Sorry, the link copy/pasted is incorrect which caused confusion.

I don't use EthrDid to resolve the domain. Is this function below incorrect?

```
    getAccountAndNetwork(provider).then(([address, chainId]) => {
      const didResolver = new Resolver(getResolver(resolverProviderConfig))

      const did = createDidFormat(address, chainId, true)
      didResolver.resolve(did).then((data: DIDDocument) => resolve(dispatch(resolveDid({ data }))))
    })
```

This should be the correct URL: https://github.com/rsksmart/rif-identity-manager/pull/5/files#diff-156fa126352dd17b9b3813d3ed3840533b4201af079451c3702461bc9d41f667R38-R43

Reply:
Let's leave it like this now. I will open an issue to find the correct solution for this :)